labs/b1.md

```diff
@@ -101,12 +95,12 @@ verify your conjectures.
 
 3. What is the root directory in Linux filesystems?
 
-4. I want `ls` to do the following for me: Give me extra information such as
+4. `ls` has a lot of cool arguments. Try using them to get extra information such as
    file permissions, owner, group, size, and late date edited. In addition, I
```

Comment:
... such as file permissions, owner name, owner group, file size, and last modified time.